src/quo/components/wallet/account_permissions/schema.cljs

```diff
@@ -0,0 +1,22 @@
+(ns quo.components.wallet.account-permissions.schema
+  (:require [quo.components.wallet.required-tokens.view :as required-tokens]))
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map
+      [:account
+       [:map
+        [:name [:maybe :string]]
+        [:address [:maybe :string]]
+        [:emoji [:maybe :string]]
+        [:customization-color [:maybe [:or :string :keyword]]]]]
```

Comment:
Are these keys always going to be passed and sometimes may be (explicitly) `nil` or are they optional? If so, should add `{:optional true}`

Reply:
Done.